docs/pages/2020_Intro_Week/sections/workflows_basic.rst

```diff
@@ -7,14 +7,15 @@ Workflows: Basics
 .. important::
 
     In order to launch the workflows of this section, we will be using the computers and codes set up in the first two hands-on sessions.
-    You should ensure then that your defalt profile is now ``quicksetup``, which we created and added to during these sessions (see :ref:`2020_virtual_intro:setup_profile`).
+    You should make sure that your default profile is set to the profile which you set up during these sessions (see :ref:`2020_virtual_intro:setup_profile`).
     You can do this using:
 
     .. code-block:: console
 
-        $ verdi profile setdefault quicksetup
-        Success: quicksetup set as default profile
+        $ verdi profile setdefault <PROFILE_NAME>
+        Success: <PROFILE_NAME> set as default profile
 
+    Where ``<PROFILE_NAME>`` is the name of the profile you set up (``quicksetup`` by default).
```

Comment:
you changed it back lol. I think they have already learnt about profiles by now, so don't need this hand holding, but its no problem

Reply:
A lot of people haven't used `quicksetup` for the profile. Maybe it's a bit hand-holdy, but I think its better to be more general. :)